src/backend/distributed/executor/executor.c

```diff
@@ -1664,9 +1679,6 @@ PlacementAccessListForTask(Task *task, ShardPlacement *taskPlacement)
 		accessType = PLACEMENT_ACCESS_SELECT;
```

Comment:
What is an example of `SELECT` queries that does not touch any shard placements?

Reply:
Reading from intermediate results could be an example:

```SQL
SELECT * FROM (SELECT * FROM users_table OFFSET 0) as foo LIMIT 1;
```